src/network/networkd-network.c

```diff
@@ -753,12 +753,13 @@ int config_parse_domains(
                 void *data,
                 void *userdata) {
 
-        Network *n = data;
+        Network *n = userdata;
```

Comment:
Was this change from `data` to `userdata` intended? Same below

Reply:
Yes. But this does not change anything, as in this case `data` and `userdata` are equivalent. The offset field for the setting in networkd-network-gperf.gperf is zero.

Reply:
got it, thanks